[2025-09-06T00:28:23.777+0000] {processor.py:186} INFO - Started process (PID=23966) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T00:28:23.778+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T00:28:23.781+0000] {logging_mixin.py:190} INFO - [2025-09-06T00:28:23.780+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T00:28:24.156+0000] {logging_mixin.py:190} INFO - [2025-09-06T00:28:24.154+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T00:28:24.156+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T00:28:24.166+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.392 seconds
[2025-09-06T01:30:50.626+0000] {processor.py:186} INFO - Started process (PID=24009) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T01:30:50.626+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T01:30:50.628+0000] {logging_mixin.py:190} INFO - [2025-09-06T01:30:50.628+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T01:30:50.967+0000] {logging_mixin.py:190} INFO - [2025-09-06T01:30:50.966+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T01:30:50.968+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T01:30:50.976+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.355 seconds
[2025-09-06T01:31:21.699+0000] {processor.py:186} INFO - Started process (PID=24052) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T01:31:21.699+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T01:31:21.701+0000] {logging_mixin.py:190} INFO - [2025-09-06T01:31:21.701+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T01:31:22.019+0000] {logging_mixin.py:190} INFO - [2025-09-06T01:31:22.018+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T01:31:22.020+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T01:31:22.028+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.333 seconds
[2025-09-06T02:52:18.954+0000] {processor.py:186} INFO - Started process (PID=24095) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T02:52:18.955+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T02:52:18.957+0000] {logging_mixin.py:190} INFO - [2025-09-06T02:52:18.957+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T02:52:19.415+0000] {logging_mixin.py:190} INFO - [2025-09-06T02:52:19.413+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T02:52:19.415+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T02:52:19.422+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.471 seconds
[2025-09-06T03:31:57.654+0000] {processor.py:186} INFO - Started process (PID=24138) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T03:31:57.656+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T03:31:57.657+0000] {logging_mixin.py:190} INFO - [2025-09-06T03:31:57.657+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T03:31:58.050+0000] {logging_mixin.py:190} INFO - [2025-09-06T03:31:58.048+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T03:31:58.050+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T03:31:58.057+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.407 seconds
[2025-09-06T03:50:16.124+0000] {processor.py:186} INFO - Started process (PID=24181) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T03:50:16.125+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T03:50:16.127+0000] {logging_mixin.py:190} INFO - [2025-09-06T03:50:16.126+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T03:50:16.549+0000] {logging_mixin.py:190} INFO - [2025-09-06T03:50:16.548+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T03:50:16.549+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T03:50:16.556+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.436 seconds
[2025-09-06T04:32:55.193+0000] {processor.py:186} INFO - Started process (PID=24224) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T04:32:55.194+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T04:32:55.196+0000] {logging_mixin.py:190} INFO - [2025-09-06T04:32:55.195+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T04:32:55.586+0000] {logging_mixin.py:190} INFO - [2025-09-06T04:32:55.585+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T04:32:55.586+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T04:32:55.593+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.403 seconds
[2025-09-06T04:33:25.846+0000] {processor.py:186} INFO - Started process (PID=24269) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T04:33:25.847+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T04:33:25.848+0000] {logging_mixin.py:190} INFO - [2025-09-06T04:33:25.848+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T04:33:26.833+0000] {logging_mixin.py:190} INFO - [2025-09-06T04:33:26.832+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T04:33:26.833+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T04:33:26.840+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.998 seconds
[2025-09-06T05:33:53.566+0000] {processor.py:186} INFO - Started process (PID=24310) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T05:33:53.567+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T05:33:53.568+0000] {logging_mixin.py:190} INFO - [2025-09-06T05:33:53.568+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T05:33:53.957+0000] {logging_mixin.py:190} INFO - [2025-09-06T05:33:53.955+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T05:33:53.957+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T05:33:53.964+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.402 seconds
[2025-09-06T05:34:24.774+0000] {processor.py:186} INFO - Started process (PID=24353) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T05:34:24.775+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T05:34:24.777+0000] {logging_mixin.py:190} INFO - [2025-09-06T05:34:24.777+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T05:34:25.184+0000] {logging_mixin.py:190} INFO - [2025-09-06T05:34:25.183+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T05:34:25.184+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T05:34:25.192+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.421 seconds
[2025-09-06T06:34:50.879+0000] {processor.py:186} INFO - Started process (PID=24396) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:34:50.880+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T06:34:50.882+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:34:50.882+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:34:51.393+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:34:51.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T06:34:51.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:34:51.402+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.526 seconds
[2025-09-06T06:35:21.619+0000] {processor.py:186} INFO - Started process (PID=24439) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:35:21.619+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T06:35:21.621+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:35:21.620+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:35:22.008+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:35:22.007+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T06:35:22.009+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:35:22.016+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.400 seconds
[2025-09-06T06:38:03.765+0000] {processor.py:186} INFO - Started process (PID=24484) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:38:03.765+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T06:38:03.766+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:38:03.766+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:38:04.168+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:38:04.166+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T06:38:04.168+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:38:04.176+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.414 seconds
[2025-09-06T06:54:58.341+0000] {processor.py:186} INFO - Started process (PID=24525) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:54:58.342+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T06:54:58.344+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:54:58.343+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:54:58.749+0000] {logging_mixin.py:190} INFO - [2025-09-06T06:54:58.747+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T06:54:58.749+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T06:54:58.756+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.418 seconds
[2025-09-06T07:36:01.696+0000] {processor.py:186} INFO - Started process (PID=24568) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T07:36:01.697+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T07:36:01.701+0000] {logging_mixin.py:190} INFO - [2025-09-06T07:36:01.701+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T07:36:02.036+0000] {logging_mixin.py:190} INFO - [2025-09-06T07:36:02.034+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T07:36:02.037+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T07:36:02.048+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.356 seconds
[2025-09-06T07:53:03.763+0000] {processor.py:186} INFO - Started process (PID=24611) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T07:53:03.763+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T07:53:03.765+0000] {logging_mixin.py:190} INFO - [2025-09-06T07:53:03.764+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T07:53:04.101+0000] {logging_mixin.py:190} INFO - [2025-09-06T07:53:04.099+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T07:53:04.101+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T07:53:04.110+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.351 seconds
[2025-09-06T08:36:59.809+0000] {processor.py:186} INFO - Started process (PID=24654) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T08:36:59.810+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T08:36:59.811+0000] {logging_mixin.py:190} INFO - [2025-09-06T08:36:59.811+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T08:37:00.129+0000] {logging_mixin.py:190} INFO - [2025-09-06T08:37:00.128+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T08:37:00.130+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T08:37:00.137+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.332 seconds
[2025-09-06T08:37:31.212+0000] {processor.py:186} INFO - Started process (PID=24697) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T08:37:31.213+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T08:37:31.215+0000] {logging_mixin.py:190} INFO - [2025-09-06T08:37:31.214+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T08:37:31.543+0000] {logging_mixin.py:190} INFO - [2025-09-06T08:37:31.541+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T08:37:31.543+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T08:37:31.553+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.346 seconds
[2025-09-06T09:37:57.321+0000] {processor.py:186} INFO - Started process (PID=24740) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T09:37:57.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T09:37:57.324+0000] {logging_mixin.py:190} INFO - [2025-09-06T09:37:57.323+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T09:37:57.664+0000] {logging_mixin.py:190} INFO - [2025-09-06T09:37:57.663+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T09:37:57.665+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T09:37:57.676+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.358 seconds
[2025-09-06T10:38:55.891+0000] {processor.py:186} INFO - Started process (PID=24783) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T10:38:55.893+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T10:38:55.895+0000] {logging_mixin.py:190} INFO - [2025-09-06T10:38:55.895+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T10:38:56.209+0000] {logging_mixin.py:190} INFO - [2025-09-06T10:38:56.207+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T10:38:56.209+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T10:38:56.217+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.329 seconds
[2025-09-06T10:39:26.965+0000] {processor.py:186} INFO - Started process (PID=24826) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T10:39:26.966+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T10:39:26.967+0000] {logging_mixin.py:190} INFO - [2025-09-06T10:39:26.967+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T10:39:27.285+0000] {logging_mixin.py:190} INFO - [2025-09-06T10:39:27.284+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T10:39:27.285+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T10:39:27.293+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.331 seconds
[2025-09-06T12:04:59.925+0000] {processor.py:186} INFO - Started process (PID=24871) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T12:04:59.926+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T12:04:59.927+0000] {logging_mixin.py:190} INFO - [2025-09-06T12:04:59.927+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T12:05:00.254+0000] {logging_mixin.py:190} INFO - [2025-09-06T12:05:00.253+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T12:05:00.255+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T12:05:00.264+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.342 seconds
[2025-09-06T13:28:16.370+0000] {processor.py:186} INFO - Started process (PID=24914) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T13:28:16.370+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T13:28:16.371+0000] {logging_mixin.py:190} INFO - [2025-09-06T13:28:16.371+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T13:28:16.682+0000] {logging_mixin.py:190} INFO - [2025-09-06T13:28:16.681+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T13:28:16.683+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T13:28:16.691+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.324 seconds
[2025-09-06T14:31:21.353+0000] {processor.py:186} INFO - Started process (PID=24955) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:31:21.356+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T14:31:21.357+0000] {logging_mixin.py:190} INFO - [2025-09-06T14:31:21.357+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:31:21.709+0000] {logging_mixin.py:190} INFO - [2025-09-06T14:31:21.708+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T14:31:21.710+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:31:21.718+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.370 seconds
[2025-09-06T14:31:51.936+0000] {processor.py:186} INFO - Started process (PID=24998) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:31:51.937+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T14:31:51.940+0000] {logging_mixin.py:190} INFO - [2025-09-06T14:31:51.940+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:31:52.388+0000] {logging_mixin.py:190} INFO - [2025-09-06T14:31:52.387+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T14:31:52.388+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:31:52.395+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.462 seconds
[2025-09-06T14:51:01.724+0000] {processor.py:186} INFO - Started process (PID=25041) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:51:01.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T14:51:01.727+0000] {logging_mixin.py:190} INFO - [2025-09-06T14:51:01.727+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:51:02.133+0000] {logging_mixin.py:190} INFO - [2025-09-06T14:51:02.132+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T14:51:02.134+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T14:51:02.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.419 seconds
[2025-09-06T15:08:26.733+0000] {processor.py:186} INFO - Started process (PID=25084) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:08:26.734+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T15:08:26.735+0000] {logging_mixin.py:190} INFO - [2025-09-06T15:08:26.735+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:08:27.132+0000] {logging_mixin.py:190} INFO - [2025-09-06T15:08:27.131+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T15:08:27.133+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:08:27.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.410 seconds
[2025-09-06T15:34:17.513+0000] {processor.py:186} INFO - Started process (PID=25127) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:34:17.514+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T15:34:17.516+0000] {logging_mixin.py:190} INFO - [2025-09-06T15:34:17.515+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:34:17.910+0000] {logging_mixin.py:190} INFO - [2025-09-06T15:34:17.909+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T15:34:17.910+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:34:17.917+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.408 seconds
[2025-09-06T15:51:30.833+0000] {processor.py:186} INFO - Started process (PID=25170) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:51:30.834+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T15:51:30.836+0000] {logging_mixin.py:190} INFO - [2025-09-06T15:51:30.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:51:31.433+0000] {logging_mixin.py:190} INFO - [2025-09-06T15:51:31.431+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T15:51:31.434+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T15:51:31.442+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.613 seconds
[2025-09-06T16:32:28.605+0000] {processor.py:186} INFO - Started process (PID=25213) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T16:32:28.606+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T16:32:28.609+0000] {logging_mixin.py:190} INFO - [2025-09-06T16:32:28.609+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T16:32:29.000+0000] {logging_mixin.py:190} INFO - [2025-09-06T16:32:28.999+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T16:32:29.000+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T16:32:29.007+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.406 seconds
[2025-09-06T16:48:37.719+0000] {processor.py:186} INFO - Started process (PID=25256) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T16:48:37.720+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T16:48:37.722+0000] {logging_mixin.py:190} INFO - [2025-09-06T16:48:37.721+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T16:48:38.138+0000] {logging_mixin.py:190} INFO - [2025-09-06T16:48:38.137+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T16:48:38.139+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T16:48:38.146+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.430 seconds
[2025-09-06T18:11:42.018+0000] {processor.py:186} INFO - Started process (PID=25301) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T18:11:42.019+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T18:11:42.020+0000] {logging_mixin.py:190} INFO - [2025-09-06T18:11:42.020+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T18:11:42.649+0000] {logging_mixin.py:190} INFO - [2025-09-06T18:11:42.647+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T18:11:42.650+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T18:11:42.658+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.644 seconds
[2025-09-06T19:50:20.067+0000] {processor.py:186} INFO - Started process (PID=25342) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T19:50:20.068+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T19:50:20.069+0000] {logging_mixin.py:190} INFO - [2025-09-06T19:50:20.069+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T19:50:20.523+0000] {logging_mixin.py:190} INFO - [2025-09-06T19:50:20.522+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T19:50:20.523+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T19:50:20.530+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.466 seconds
[2025-09-06T21:12:25.601+0000] {processor.py:186} INFO - Started process (PID=25385) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T21:12:25.602+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T21:12:25.603+0000] {logging_mixin.py:190} INFO - [2025-09-06T21:12:25.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T21:12:26.001+0000] {logging_mixin.py:190} INFO - [2025-09-06T21:12:25.999+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T21:12:26.001+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T21:12:26.008+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.411 seconds
[2025-09-06T22:50:02.155+0000] {processor.py:186} INFO - Started process (PID=25430) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T22:50:02.156+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T22:50:02.157+0000] {logging_mixin.py:190} INFO - [2025-09-06T22:50:02.157+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T22:50:02.494+0000] {logging_mixin.py:190} INFO - [2025-09-06T22:50:02.493+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T22:50:02.495+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T22:50:02.503+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.351 seconds
[2025-09-06T23:04:10.851+0000] {processor.py:186} INFO - Started process (PID=25471) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T23:04:10.852+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T23:04:10.854+0000] {logging_mixin.py:190} INFO - [2025-09-06T23:04:10.854+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T23:04:11.196+0000] {logging_mixin.py:190} INFO - [2025-09-06T23:04:11.194+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T23:04:11.196+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T23:04:11.205+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.357 seconds
[2025-09-06T23:37:28.084+0000] {processor.py:186} INFO - Started process (PID=25514) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T23:37:28.084+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-06T23:37:28.086+0000] {logging_mixin.py:190} INFO - [2025-09-06T23:37:28.086+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T23:37:28.438+0000] {logging_mixin.py:190} INFO - [2025-09-06T23:37:28.436+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-06T23:37:28.438+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-06T23:37:28.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.367 seconds
